client/client.go

```diff
@@ -238,23 +205,24 @@ func (pb *ProofBuilder) InclusionProof(ctx context.Context, index uint64) ([][]b
 
 	span.SetAttributes(indexKey.Int64(otel.Clamp64(index)))
 
-	nodes, err := proof.Inclusion(index, pb.cp.Size)
+	nodes, err := proof.Inclusion(index, pb.treeSize)
 	if err != nil {
 		return nil, fmt.Errorf("failed to calculate inclusion proof node list: %v", err)
 	}
 	return pb.fetchNodes(ctx, nodes)
 }
 
-// ConsistencyProof constructs a consistency proof between the two passed in tree sizes.
+// ConsistencyProof constructs a consistency proof between the smaller passed-in treesize, and
```

Comment:
```suggestion
// ConsistencyProof constructs a consistency proof between the smaller passed-in tree size, and
```

Reply:
(I decided to leave the params here as they were, and pass in both `smaller` and `larger` sizes explicitly for now)

Reply:
Gotcha. Shall we use `treeSize` for param?